Automattic/wp-calypso
# Plans overhaul: Add Google Analytics for Starter

#### Changes proposed in this Pull Request

1. Adds the Google Analytics feature to the plans grid
2. Adds the Google Analytics feature block to `plans/my-plan`
3. Adds the Google Analytics feature block to the checkout thank-you page

(this will also be included in the about-to-be-obsolete "free vs pro" grid, but I guess that's fine - it will just be marked with "not included")

#### Media

`start/plans`

<img width="500" alt="Screenshot 2022-05-18 at 4 37 24 PM" src="https://user-images.githubusercontent.com/1705499/169052243-3e502307-d2a7-47eb-9641-7613abcd92b9.png">

`plans/my-plan`

<img width="500" alt="Screenshot 2022-05-18 at 4 37 55 PM" src="https://user-images.githubusercontent.com/1705499/169052272-e0324a67-54e3-4a34-97ec-4c50745d7594.png">


#### Testing instructions



* Go to `/start/plans` and select "show full plan comparison"
    * confirm feature is included in list as in media above
* Go to `/plans/[SITE_ON_FREE_OR_STARTER]` and select "show full plan comparison"
    * confirm feature is included in list as in media above
* Go to `/plans/my-plan/[SITE_ON_STARTER]` and confirm a GA block is there
    * Click on it and confirm the form that opens is working as expected (the backend parts should be in place)
* Go to `/start/plans` and purchase a Starter plan. Complete the checkout
    * Confirm the "thank you" page includes a block with GA info (same as with Pro)
* Disable the flag (e.g. `/start/plans?flags=-plans/starter-plan`) and ensure nothing breaks. 
    * The GA feature should still be in the list, and marked as "not included" for the free plan. (assuming this is ok)



#### TODO

- [ ] Let's reorder the features list and make any other contextual updates to the list in this PR. Not sure if the final order has been settled yet. See Slack thread: p1652878563789739/1652865158.458459-slack-C031TFM2NKC
- [ ] Let's update the subtitle of the Starter plan also once it's finalized/settled. Same Slack thread: p1652878563789739/1652865158.458459-slack-C031TFM2NKC

Related to Automattic/martech#741
Fixes Automattic/martech#741
